protocols/grpc/hyperstart.proto

```diff
@@ -37,9 +37,10 @@ service HyperstartService {
 
 message CreateContainerRequest {
 	string container_id = 1;
-	StringUser string_user = 2;
-	repeated Storage storages =3;
-	Spec OCI = 4;
+	string process_id = 2;
```

Comment:
Would this `process_id` be needed for signaling the container init process later on?